### PR TITLE
Add uprobe/uretprobe wildcard matching

### LINKED ISSUE #67: uprobe/uretprobe to support wildcards
```
# bpftrace -e 'uprobe:/lib/x86_64-linux-gnu/libc.so.6:gethostbyname* { printf("got: %s\n", str(arg0)); }'
Wildcard matches aren't available on probe type 'uprobe'
No probes to attach
```

Works with kprobes.

### PR DESCRIPTION
**Pull request by acj**
I mostly use bpftrace with Go programs, and it would be very helpful to have wildcard support, e.g. for attaching uprobes without having specify the entire package path. This is a learning PR, so I'll be grateful for any constructive feedback.

Resolves #67.

### CODE REVIEW

File: src/bpftrace.cpp [hunk 1]
```diff
@@ -1445,6 +1467,15 @@ uint64_t BPFtrace::resolve_uname(const std::string &name, const std::string &pat
   return addr;
 }
 
+std::string BPFtrace::extract_func_symbols_from_path(const std::string &path)
+{
+  std::string call_str = std::string("objdump -tT ") + path +
```
Comment (brendangregg) [inline src/bpftrace.cpp]: It's minor, but I'd have a comment above this that said something like "// XXX Shell out to objdump for now; in the future, replace with library calls.".

Just so we remember to one day explore using libelf or whatever for this.

**Comment by brendangregg** [comment]
Looks good to me. When you're ready I'll do a squash-merge.

File: src/bpftrace.cpp (hunk 1 above)
Comment (acj) [reply src/bpftrace.cpp]: The function above has a TODO item for that very issue, so I copied it into this one.

**Comment by brendangregg** [comment]
Thanks!

**Pull request merged**